tests/ui/conftest.py

```diff
@@ -100,12 +101,17 @@ def user(
     }
     assert requests.codes.created == response.status_code
     user.update(response.json())
-    print(user)
     return user
 
 
 @pytest.fixture(scope='function')
-def our_live_server(request):
+def live_server(request):
+    """
+        This fixture overrides the live_server fixture provided by
+        pytest_django. live_server allows us to create a running version of the
+        addons django application within pytest for testing.
```

Comment:
I'm still not understanding why we need to override this fixture. It looks like we could just use it unmodified, after reading the source code for the version in use: https://github.com/pytest-dev/pytest-django/blob/2.9.1/pytest_django/fixtures.py#L295-L330